README.md

```diff
@@ -194,6 +194,7 @@ The following are optional as `step.with` keys
 | `generate_release_notes`   | Boolean | Whether to automatically generate the name and body for this release. If name is specified, the specified name will be used; otherwise, a name will be automatically generated. If body is specified, the body will be pre-pended to the automatically generated notes. See the [GitHub docs for this feature](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes) for more information |
 | `append_body`              | Boolean | Append to existing body instead of overwriting it                                                                                                                                                                                                                                                                                                                                                                                               |
 | `make_latest`              | String  | Specifies whether this release should be set as the latest release for the repository. Drafts and prereleases cannot be set as latest. Can be `true`, `false`, or `legacy`. Uses GitHub api defaults if not provided                                                                                                                                                                                                                            |
+| `update_tag`               | Boolean | Update the tag of the release to the current commit. This will also update the release time. Default is false                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
I have some reservations about this. Action gh release is an action for managing GitHub releases, the product feature

the concern is that with these changes the scope is increasing to also include managing git state something that feels possible with git itself external to this action. 

Would it be possible to do that as a separate workflow step or possibly trigger that then triggers the workflow that manages gh release data?